src/APICore.ts

```diff
@@ -64,35 +64,7 @@ export default class API {
     }
 
     async get<T = Record<string, unknown>>(url: string, args?: CoveoPlatformClientRequestInit): Promise<T> {
-        try {
-            return await this.request<T>(url, this.buildRequestInit('GET', args), args?.responseBodyFormat);
-        } catch (error) {
-            // Warning: the logic below does not do what the original author intended/mentions.
-            // It will fullfil the promise with `undefined`, instead of keeping it pending.
-            if (error.name === 'AbortError') {
```

Comment:
Is there any documentation in the main README.md of the package to explain how the user can do this? Finally, you've explained a good solution in the description of your pull request. I think you can provide the same solution in the readme, so future users will be happy to have your solution. What do you think?

Reply:
We can add it in the documentation, but at the same time this is kind of a bad practice that a user should avoid. I can add the example to the changelog in the github release it self. This way if someone is looking for a migration path it will be there.